cg/meta/orders/ticket_handler.py

```diff
@@ -70,29 +72,29 @@ def create_ticket(
 
         return ticket_response.id
 
-    def create_attachment_file(self, order: OrderIn, temp_dir: str) -> Path:
+    def create_attachment_file(self, order: Order, temp_dir: str) -> Path:
         """Create a single attachment file for the ticket"""
         order_file_path = Path(temp_dir) / "order.json"
         with order_file_path.open("w") as order_file:
             order_file.write(order.json())
         return order_file_path
 
-    def create_xml_sample_list(self, order: OrderIn, user_name: str) -> str:
+    def create_xml_sample_list(self, order: Order, user_name: str) -> str:
         message = ""
-        for sample in order.samples:
-            message = self.add_sample_name_to_message(message=message, sample_name=sample.name)
-            message = self.add_sample_apptag_to_message(
-                message=message, application=sample.application
-            )
-            if isinstance(sample, Of1508Sample):
-                message = self.add_sample_case_name_to_message(
-                    message=message, case_name=sample.family_name
+        if isinstance(order, OrderWithCases):
+            message = self.create_case_xml_sample_list(order=order, message=message)
```

Comment:
```suggestion
            message = self.add_existing_sample_info_to_message
```
Should be this method right?